packages/wdio-shadow-dom-service/tests/service.test.js

```diff
@@ -0,0 +1,89 @@
+import ShadowDomService from '../src'
+
+describe('ShadowDomService', () => {
+
+    let service
+    beforeAll(() => {
```

Comment:
We will want to break this up into 2 different unit tests (probably browser and service).

Reply:
The only thing the package provides is a service. It could be broken into multiple tests I guess... one for the service bits that add the commands, and another for the commands themselves.